contents/updates/content-updates.html

```diff
@@ -0,0 +1,45 @@
+---
+title: Updates
+layout: layout-post
+permalink: /updates/
+---
+
+<h2>Migrating to v2</h2>
+<h6>Last updated on July 14th, 2022</h6>
+<p>
+  Yes, we are finally migrating to a new portal soon! The v2 will be released by
```

Comment:
Change this to "We will be upgrading to a new portal soon with a new base framework.